clang/include/clang/Basic/TargetInfo.h

```diff
@@ -14,6 +14,7 @@
 #ifndef LLVM_CLANG_BASIC_TARGETINFO_H
 #define LLVM_CLANG_BASIC_TARGETINFO_H
 
+#include "clang/AST/Attr.h"
```

Comment:
Basic shouldn't be referencing AST (IIRC, it shouldn't reference anything but basic or llvm), this ends up being an unfortunate layering violation.